torch/ao/nn/quantized/dynamic/modules/rnn.py

```diff
@@ -610,6 +610,26 @@ class GRU(RNNBase):
         All the weights and biases are initialized from :math:`\mathcal{U}(-\sqrt{k}, \sqrt{k})`
         where :math:`k = \frac{1}{\text{hidden\_size}}`
 
+    .. note::
+        The calculation of new gate :math:`n_t` subtly differs from the orignal paper and other frameworks.
```

Comment:
nit, and same below in torch.nn.GRU
```suggestion
        The calculation of new gate :math:`n_t` subtly differs from the original paper and other frameworks.
```